app/models/subscriber_list.rb

```diff
@@ -16,6 +16,12 @@ class SubscriberList < ApplicationRecord
   has_many :matched_content_changes
   has_many :matched_messages
 
+  before_validation do
```

Comment:
I'd suggest using a before_save block here rather than validation since we don't use this data in validation